bundle/language_ar.properties

```diff
@@ -314,6 +314,8 @@ angal.patient.nextkin                                                       = ذ
 angal.patient.note                                                          = ملاحظة
 angal.patient.parenttogether                                                = الآباء معا
 angal.patient.patientphoto                                                  = صورة المريض
+angal.patient.patientphoto.error.title                                      = 
+angal.patient.patientphoto.error.filenotfound                               = 
```

Comment:
There's no need to add new keys in the language other than "en". Please revert this change.